test/config/integration/certs/certs.sh

```diff
@@ -2,32 +2,32 @@
 
 set -e
 
-# Uncomment the following lines if you want to regenerate the private keys.
```

Comment:
There isn't much point in regenerating private keys, why do you want to do that?

Reply:
To prevent bit rot, e.g. let's say some test starts to take some dependency on an attribute of the key (size, hash, etc.). We should move to a model closer to that where we would be doing this on every test invocation.

Reply:
I don't think that we have anything that relies on private key's attributes, but I don't have a strong reason against it either.